pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -275,6 +368,123 @@ def test_comparisons_coverage(self):
 # Timedelta64[ns] dtype Arithmetic Operations
 
 
+@given(positive_td=positive_tds)
+def test_add_raises_expected_error_if_result_would_overflow(
+    add_sub_types: BinaryOpTypes,
+    positive_td: Timedelta,
+):
+    left = wrap_value(Timedelta.max, add_sub_types.left)
+    right = wrap_value(positive_td, add_sub_types.right)
+
+    if add_sub_types.result is Timedelta:
```

Comment:
Should this be raising a `OutOfBoundsTimedelta` in all cases? Ref: https://github.com/pandas-dev/pandas/pull/34448

Reply:
probably, yes